cocos2d/core/platform/CCConfig.js

```diff
@@ -56,7 +56,7 @@ window["CocosEngine"] = cc.ENGINE_VERSION = "Cocos2d-JS v3.11";
  * @constant
  * @type {Number}
  */
-cc.FIX_ARTIFACTS_BY_STRECHING_TEXEL = 0;
+cc.FIX_ARTIFACTS_BY_STRECHING_TEXEL = 1;
```

Comment:
Ok, this was copied from @Heishe 's PR
This is for removing the line at the border of texture
I think I can keep it the old way